src/content/docs/agents/net-agent/attributes/enable-disable-attributes-net.mdx

```diff
@@ -211,32 +211,33 @@ New Relic follows these rules to determine which attributes to include or exclud
       <include>foo</include>
     </attributes>
     <transactionEvents enabled="true">
-      <attributes enabled=”true”>
+      <attributes enabled="true">
         <exclude>foo</exclude>
+      </attributes>
     </transactionEvents>
     ```
 
     Example output:
 
     ```
     Keys passed in: foo
-    Keys included for transaction events: 
+    Keys included for transaction events:
     Keys included for other destinations: foo
     Keys excluded for transaction events: foo
     ```
   </Collapser>
 </CollapserGroup>
 
-## Deprecated properties [#deprecated]
+## Obsolete properties [#obsolete]
 
-The following properties have been deprecated. While attributes using these properties will continue to be collected accurately, New Relic recommends switching to the new attributes configuration for these properties when upgrading your .NET agent.
+The following properties are not available in the .NET agent v9.0. Please visit the [.NET agent 8.x to 9.x migration guide](/docs/agents/net-agent/getting-started/8x-to-9x-agent-migration-guide) page for replacement properties when upgrading your .NET agent.
```

Comment:
This is great, thanks for pointing to the migration guide! And this just reminded me to update the callout at the top of the migration guide itself. (I wouldn't have expected you to know about that.) I'll do that in my own branch so you can proceed with this one.